OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -592,6 +676,11 @@ function Test-TargetResource {
             }
         }
     }
+    else
+    {
+        # Display the machine is not registered
+        Write-Verbose "Space $Space does not have a registration for $Name"
```

Comment:
This is matched up with the `if ($Ensure -eq "Present" -and ![string]::IsNullOrEmpty($OctopusServerUrl))` line - it's not the else case for a "machine is not registered in space"?

```suggestion
        Write-Verbose "Machine `$Name` is not registered in Space `$Space`"
```